script.js

```diff
@@ -0,0 +1,229 @@
+
+const TEXTAREA = document.createElement('textarea');
+TEXTAREA.id = 'textarea';
+TEXTAREA.autofocus = true;
+const KEYBOARD = document.createElement('div');
```

Comment:
I think for such contants it is better to use camelCase
`const keyboard = document.createElement('div');`
Upper case is commonly used for unchangable things
`const MESSAGE = 'Shift + Alt - change language\n Created under Windows'`

Reply:
fix in commit 963f612